From c09e26be8ece49d7230f4e94422b2fc50e164c13 Mon Sep 17 00:00:00 2001
From: Thomas Koch <linrunner@gmx.net>
Date: Sun, 2 Feb 2020 14:45:45 +0100
Subject: [PATCH] Issue #460: default CPU_ENERGY_PERF_POLICY_ON_BAT=power too
 aggressive

Confirmed for:
* ThinkPad T480s, i7 8550U, Linux 5.4
* ThinkPad X1C6,  i7 8550U, Linux 5.5

Solution: change default to balance_power.

Reference:
* https://github.com/linrunner/TLP/issues/460
---
 VERSION       | 2 +-
 changelog     | 6 ++++++
 defaults.conf | 2 +-
 tlp.conf      | 4 ++--
 4 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/VERSION b/VERSION
index f0bb29e..3a3cd8c 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-1.3.0
+1.3.1
diff --git a/changelog b/changelog
index f0afff6..8054928 100644
--- a/changelog
+++ b/changelog
@@ -1,3 +1,9 @@
++++ 1.3.1 --- <unreleased> +++
+
+  * Bugfixes
+    Processor:
+      - Issue #460: default CPU_ENERGY_PERF_POLICY_ON_BAT=power too aggressive
+
 +++ 1.3.0 --- 31.01.2020 +++
 
   * Features
diff --git a/defaults.conf b/defaults.conf
index da3e07b..786266d 100644
--- a/defaults.conf
+++ b/defaults.conf
@@ -8,7 +8,7 @@ DISK_IDLE_SECS_ON_BAT=2
 MAX_LOST_WORK_SECS_ON_AC=15
 MAX_LOST_WORK_SECS_ON_BAT=60
 CPU_ENERGY_PERF_POLICY_ON_AC=balance_performance
-CPU_ENERGY_PERF_POLICY_ON_BAT=power
+CPU_ENERGY_PERF_POLICY_ON_BAT=balance_power
 SCHED_POWERSAVE_ON_AC=0
 SCHED_POWERSAVE_ON_BAT=1
 NMI_WATCHDOG=0
diff --git a/tlp.conf b/tlp.conf
index 7162e15..389a788 100644
--- a/tlp.conf
+++ b/tlp.conf
@@ -95,10 +95,10 @@
 # - EPB requires kernel 5.2 or module msr and x86_energy_perf_policy
 #   from linux-tools
 # - When HWP.EPP is available, EPB is not set
-# Default: balance_performance (AC), power (BAT)
+# Default: balance_performance (AC), balance_power (BAT)
 
 #CPU_ENERGY_PERF_POLICY_ON_AC=balance_performance
-#CPU_ENERGY_PERF_POLICY_ON_BAT=power
+#CPU_ENERGY_PERF_POLICY_ON_BAT=balance_power
 
 # Set Intel CPU P-state performance: 0..100 (%).
 # Limit the max/min P-state to control the power dissipation of the CPU.
-- 
2.25.0.2

